### ISSUE TITLE
Use abbr element for acronyms

### ISSUE THREAD

**Issue by mgifford** [18F/508-procurement-playbook#17]
This document should be an example of good accessibility.  Lets make sure to define acronyms properly:
https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140916/H28

I don't know what a CLINs is but assume it might be `<abbr title="contract line item numbers ">CLIN</abbr>`s
